src/assets/css/_sass/elements/_buttons.scss

```diff
@@ -5,3 +5,23 @@
   Don't forget hover, active states
 
 */
+
+.button {
+  padding: .575rem .75rem;
+  text-decoration: none;
```

Comment:
Is this a button, or is this a link that that looks like a button. This can be moved to it's own selector for the specific case.